scripts/tasks/src/presets.ts

```diff
@@ -64,14 +65,15 @@ export function preset() {
   task('eslint', eslint);
   task('webpack', webpack);
   task('webpack-dev-server', webpackDevServer(args));
-  task('api-extractor', apiExtractor());
+  task('api-extractor', apiExtractor);
```

Comment:
improves task processing performance as apiExtractor is invoked only on call site ( previously was invoked with every `just-scripts` calls